apps/omg/lib/omg/release_tasks/init_kv_db.ex

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018-2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.ReleaseTasks.InitKVDB do
+  @moduledoc false
```

Comment:
what do you mean by that? how would you name the module? currently it means initialization of KV database

Reply:
`InitKeyValueDB`, KVDB may mean something else also,
or simple write in moduledoc `initing key value database`

Reply:
heeey, this convo caught my eye. Please call it `OMG.DB`/`omg_db` etc instead of introducing a new name (as this is what it is, correct?).

(A module doc would be also nice regardless of the name, but provide info which isn't obvious from the name, like "Creates an empty instance of `OMG.DB` storage and fills it with the required initial data")

If my ramblings are totally off, pls ignore.